packages/mdc-textfield/_mixins.scss

```diff
@@ -129,17 +129,21 @@
 // Baseline
 
 @mixin mdc-text-field-disabled_ {
-  @include mdc-text-field-bottom-line-color_($mdc-text-field-disabled-border);
+  @include mdc-text-field-bottom-line-color_($mdc-text-field-outlined-disabled-border);
   @include mdc-text-field-ink-color_($mdc-text-field-disabled-ink-color);
   @include mdc-text-field-label-ink-color_($mdc-text-field-disabled-label-color);
   @include mdc-text-field-helper-text-color_($mdc-text-field-disabled-helper-text-color);
   @include mdc-text-field-icon-color_($mdc-text-field-disabled-icon);
   @include mdc-text-field-fullwidth-bottom-line-color_($mdc-text-field-fullwidth-bottom-line-color);
+  @include mdc-text-field-fill-color_($mdc-text-field-disabled-background);
+  @include mdc-text-field-label-color($mdc-text-field-disabled-label-color);
 
+  border-bottom: none;
   pointer-events: none;
 
   .mdc-text-field__input {
-    border-bottom: 1px dotted;
+    border-bottom-width: 1px;
```

Comment:
This can also be removed, since the input already has a default border-bottom of 1px.

Reply:
Complete!